vdyp-forward/src/main/java/ca/bc/gov/nrs/vdyp/forward/VdypUtilizationParser.java

```diff
@@ -0,0 +1,145 @@
+import java.util.Map;
+import java.util.Optional;
+
+import ca.bc.gov.nrs.vdyp.common.ControlKey;
+import ca.bc.gov.nrs.vdyp.common.ValueOrMarker;
+import ca.bc.gov.nrs.vdyp.forward.model.VdypSpeciesUtilization;
+import ca.bc.gov.nrs.vdyp.io.EndOfRecord;
+import ca.bc.gov.nrs.vdyp.io.FileResolver;
+import ca.bc.gov.nrs.vdyp.io.parse.common.LineParser;
+import ca.bc.gov.nrs.vdyp.io.parse.common.ResourceParseException;
+import ca.bc.gov.nrs.vdyp.io.parse.control.ControlMapValueReplacer;
+import ca.bc.gov.nrs.vdyp.io.parse.streaming.AbstractStreamingParser;
+import ca.bc.gov.nrs.vdyp.io.parse.streaming.GroupingStreamingParser;
+import ca.bc.gov.nrs.vdyp.io.parse.streaming.StreamingParserFactory;
+import ca.bc.gov.nrs.vdyp.io.parse.value.ControlledValueParser;
+import ca.bc.gov.nrs.vdyp.io.parse.value.ValueParser;
+import ca.bc.gov.nrs.vdyp.model.LayerType;
+import ca.bc.gov.nrs.vdyp.model.UtilizationClass;
+
+public class VdypUtilizationParser
+		implements ControlMapValueReplacer<StreamingParserFactory<Collection<VdypSpeciesUtilization>>, String> {
+
+	private static final String DESCRIPTION = "DESCRIPTION"; // POLYDESC
+	private static final String LAYER_TYPE = "LAYER_TYPE"; // LAYERG
+	private static final String GENUS_INDEX = "SPECIES_INDEX"; // ISP
+	private static final String GENUS = "SPECIES"; // SP0
+	private static final String UTILIZATION_CLASS_INDEX = "UTILIZATION_CLASS_INDEX"; // J
+	private static final String BASAL_AREA = "BASAL_AREA"; // BA
+	private static final String LIVE_TREES_PER_HECTARE = "LIVE_TREES_PER_HECTARE"; // TPH
+	private static final String LOREY_HEIGHT = "LOREY_HEIGHT"; // LHJ
+	private static final String WHOLE_STEM_VOLUME = "WHOLE_STEM_VOLUME"; // VOLWS
+	private static final String CLOSE_UTIL_VOLUME = "CLOSE_UTIL_VOLUME"; // VOLCU
+	private static final String CU_VOLUME_LESS_DECAY = "CU_VOLUME_LESS_DECAY"; // VOL_D
+	private static final String CU_VOLUME_LESS_DECAY_WASTAGE = "CU_VOLUME_LESS_DECAY_WASTAGE"; // VOL_DW
+	private static final String CU_VOLUME_LESS_DECAY_WASTAGE_BREAKAGE = "CU_VOLUME_LESS_DECAY_WASTAGE_BREAKAGE"; // VOL_DWB
+	private static final String QUADRATIC_MEAN_DIAMETER_BREAST_HEIGHT = "QUADRATIC_MEAN_DIAMETER_BREAST_HEIGHT"; // DQ
+
+	@Override
+	public ControlKey getControlKey() {
+		return ControlKey.VDYP_LAYER_BY_SP0_BY_UTIL;
+	}
+
+	@Override
+	public StreamingParserFactory<Collection<VdypSpeciesUtilization>>
+			map(String fileName, FileResolver fileResolver, Map<String, Object> control)
+					throws IOException, ResourceParseException {
+		return () -> {
+			var lineParser = new LineParser().strippedString(25, DESCRIPTION).space(1)
+					.value(
+							1, LAYER_TYPE,
+							ValueParser.valueOrMarker(
+									ValueParser.LAYER,
+									ValueParser.optionalSingleton(
+											x -> x == null || x.trim().length() == 0 || x.trim().equals("Z"),
+											EndOfRecord.END_OF_RECORD
+									)
+							)
+					).value(3, GENUS_INDEX, ValueParser.INTEGER).space(1)
```

Comment:
I find using empty comments to force the formatter to retain line breaks can help readability in situations like fluent method calls.